source/administration/statistics.rst

```diff
@@ -0,0 +1,66 @@
+================
+
+Statistics on users, posts and channels are tracked for each system and team. Enterprise Editions have access to advanced system statistics.
+
+Site statistics
+-----------------
+
+System statistics are viewable under **System Console** > **Reporting** > **Site Statistics**. The data shown here is a cumulative sum
+across all teams on the system.
+
+**Total Users:** The total number of accounts created on your system, including both active and inactive accounts.
+
+**Total Teams:** The total number of teams created on your system, including deleted teams.
+
+**Total Posts:** The total number of posts made in all the teams on your system, including deleted posts and posts made using automation.
+
+**Total Channels:** The total number of channels created in all the teams on your system, including deleted channels.
+
+**Total Posts (graph):** The total number of posts made on a certain day in all the teams on your system, including deleted posts and posts made using automation.
+
+**Active Users With Posts (graph):** Users who logged in and made a post on a certain day in all the teams on your system, including system messages posted from the user's account.
+
+Advanced system statistics (Enterprise)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Enterprise Edition includes additional system statistics.
+
+**Total Sessions:** The number of active user sessions connected to your system. Expired sessions are not counted.
+
+**Total Commands:** The number of active slash commands currently setup on your system. Slash commands that are created and then removed in the Integrations menu are not counted.
+
+**Incoming Webhooks:** The number of active Incoming Webhooks currently setup on your system. Incoming webhooks that are created and then removed in the Integrations menu are not counted.
+
+**Outgoing Webhooks:** The number of active Outgoing Webhooks currently setup on your system. Outgoing webhooks that are created and then removed in the Integrations menu are not counted.
+
+**Websocket Conns:** The number of active WebSocket connections currently on your server.
+
+**Master DB Conns:** The number of active connections currently on your master database.
+
+**Replica DB Conns:** The number of active connections currently on one or more of `your read replica databases <https://docs.mattermost.com/deployment/cluster.html#database-configuration>`_.
+
+**Channel Types:** This chart displays the number of public channels and private groups in a visual format, including channels that may have been archived.
+
+**Posts, Files and Hashtags:** This chart displays the number of posts containing files, hashtags or text-only. Posts containing both files and hashtags are counted in both categories.
+
+Team statistics
+---------------
+
+Team Statistics are viewable under **System Console** > **Teams** > **Statistics**.
+
+**Total Users:** The total number of accounts created on your system, including both active and inactive accounts.
```

Comment:
Doesn't actually include inactive accounts. Also this should say something about "who are members of this team".